ioniq 5 top view card.yaml

```diff
@@ -118,24 +118,30 @@ elements:
       left: 50%
       transform: translate(-50%,-50%) scale(0.1,0.1)
   - type: custom:button-card
-    entity: sensor.ioniq_5_car_battery_level
+    entity: sensor.la_luna_car_battery_level
```

Comment:
this is only specific to you, having set this up for a few people they either have ioniq_5_car...... or 2024_ioniq_5_car....